src/Cpp/1-getting-started/1-3-4-3DRendering/3DRenderingApplication.cpp

```diff
@@ -342,13 +398,15 @@ void Rendering3DApplication::Render()
     float clearColor[] = { 0.1f, 0.1f, 0.1f, 1.0f };
     constexpr uint32_t vertexOffset = 0;
 
-    ID3D11RenderTargetView* nullTarget = nullptr;
+    ID3D11RenderTargetView* nullRTV = nullptr;
+
+    //set to nullptr so we can clear properly
```

Comment:
instead of saying `set to nullptr so we can clear properly`
write `this will ensure proper clear` (its obvious that we are setting it in the line of code below, doesn't need separate mention)